frontend/public/components/modals/language-preferences-modal.tsx

```diff
@@ -1,7 +1,8 @@
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
 import i18next, { TFunction } from 'i18next';
-
+// eslint-disable-next-line import/no-unresolved
+import { QuickStartContextValues } from '@patternfly/quickstarts';
```

Comment:
why is eslint-disable used here?

Reply:
I was getting this eslint error, only when I was pulling in a type from the lib as I am doing here. I can take another look tho to see if I can fix it without disabling for this line.

Reply:
@jschuler is this still required?